acpreprocessing/stitching_modules/convert_to_n5/tiff_to_ngff.py

```diff
@@ -471,9 +480,9 @@ def iterate_mip_levels_from_mimgfns(
                 block_offset:block_offset+chunk_size, :, :] = chunk[:, :, :]
 
             # copy op only for uneven final chunk
-            if chunk_size != block_size:
-                temp_lminus1_arr = temp_lminus1_arr[
-                    :block_offset+chunk_size, :, :]
+            # if chunk_size != block_size:
```

Comment:
Pretty sure the edge case still exists as well, it's commented out because I need to go back and revise this in the context of the deskew consequences on "blocks" and "chunks" - basically IIRC I think it was causing the script to hang indefinitely because of a mismatch.